src/util/Random.h

```diff
@@ -13,55 +13,64 @@
 #include <vector>
 
 /**
- * @brief Create random integers from a given range
- * @tparam Int an integral type like int, unsigned int, size_t, etc...
+ * A simple and fast Pseudo-Random-Number-Generator called Xoroshiro128+, for
+ * details see
+ * https://thompsonsed.co.uk/random-number-generators-for-c-performance-tested
+ * Limiting the range of the generated numbers is currently not supported
  */
```

Comment:
Missing full stop + add:

Assumes that `Int` fits into 64 bits. If used with a type that does not fulfill this property, the template will not match (because of the std::enable_if) and there will be a compile-time error.